lib/temporal/connection.rb

```diff
@@ -0,0 +1,294 @@
+require 'temporal/api/workflowservice/v1/request_response_pb'
+require 'temporal/bridge'
+
+module Temporal
+  class Connection
+    def initialize(host)
```

Comment:
I assume a future version of this will take all the other options? If so, no worries

Reply:
Yep, that's right 👍